buildSrc/src/main/groovy/airbyte-connector-acceptance-test.gradle

```diff
@@ -1,10 +1,27 @@
 import org.gradle.api.Plugin
 import org.gradle.api.Project
 
+
 class AirbyteConnectorAcceptanceTestPlugin implements Plugin<Project> {
+    
+    // Change this value to 'dev' if you want to use a locally built connector acceptance test image
+    String CONNECTOR_ACCEPTANCE_TEST_IMAGE = 'latest'
+    
     void apply(Project project) {
+        
         project.task('connectorAcceptanceTest') {
+            dependsOn project.build
+            dependsOn project.airbyteDocker
+            
             doFirst {
+                if (CONNECTOR_ACCEPTANCE_TEST_IMAGE != "dev") {
+                    project.exec {
+                        def args = [
+                            'docker', 'pull', 'airbyte/connector-acceptance-test:latest'
```

Comment:
TODO: Perhaps highly consider using a versioned image here, since currently we can publish to this a breaking change which pulls the rug out from under our test suite. I'm ok with this going in as-is for now because the speed improvements are so dramatic

Reply:
Is the base image static enough that we could go ahead and pin this today and you could remember to update it when you update the base image?

Reply:
Yep, this is something I considered.
The image is quite dynamic but mainly maintained by an identified pool of engineer I trust enough to write unit tests and locally tests their changes before publishing.
I also like that we can quickly pin the version if the latest one has a bug.

There's also no better approach than waiting for nightly builds to run for all connectors on the latest version to assess the stability of tests. If the connector tests were reasonably fast we could automatically run the branch version of connector acceptance tests on a pool of previously succeeding connectors and then autobump the versions used on master if 🟢 . But this is not something we can currently achieve.

Reply:
👍 
If you agree, a TODO in the code is sufficient for me then. Otherwise feel free to leave as-is.